### PR TITLE
CNV#30164: 4.15 CNV Networking release notes

### PR DESCRIPTION
**Pull request by sjhala-ccs**
Version(s): 4.15

Issue: [CNV-30164](https://issues.redhat.com//browse/CNV-30164) [CNV-30167](https://issues.redhat.com//browse/CNV-30167) [CNV-13680](https://issues.redhat.com//browse/CNV-13680) [CNV-30170](https://issues.redhat.com//browse/CNV-30170)

Link to docs preview: https://71842--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-15-release-notes#virt-4-15-networking

QE review:
- [x] QE has approved this change.

Additional information: This PR covers multiple CNV Networking release notes

### CODE REVIEW

File: virt/release_notes/virt-4-15-release-notes.adoc [hunk 1]
```diff
@@ -85,14 +85,14 @@ This release adds new features and enhancements related to the following compone
...
 
 //CNV-13680: Connecting OVN-Kubernetes to underlying physical networks. Remove known issue BZ-1885605 after adding this note.
+* {VirtProductName} now supports the localnet topology for xref:../../virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc#virt-connecting-vm-to-ovn-secondary-network[OVN-Kubernetes secondary networks]. A localnet topology connects the secondary network to the physical underlay. This enables both east-west cluster traffic and access to services running outside the cluster, but it requires additional configuration of the underlying Open vSwitch (OVS) system on cluster nodes.
+
+//CNV-30170: NetworkPolicies filtering using IP blocks with OVN-Kubernetes
+* An OVN-Kubernetes secondary network is compatible with the xref:../../networking/multiple_networks/configuring-additional-network.adoc#compatibility-with-multi-network-policy_configuring-additional-network[multi-network policy API] which provides the `MultiNetworkPolicy` custom resource definition (CRD) to control traffic flow to and from VMs. You can use the `ipBlock` attribute to define network policy ingress and egress rules for a specific CIDR block.
```
Comment (maiqueb) [inline virt/release_notes/virt-4-15-release-notes.adoc:95]:
```suggestion
* An OVN-Kubernetes secondary network is compatible with the xref:../../networking/multiple_networks/configuring-additional-network.adoc#compatibility-with-multi-network-policy_configuring-additional-network[multi-network policy API] which provides the `MultiNetworkPolicy` custom resource definition (CRD) to control traffic flow to and from VMs. You can use the `ipBlock` attribute to define network policy ingress and egress rules for specific CIDR blocks.
```

**Review by maiqueb — CHANGES_REQUESTED** [review]
I think plural is more correct here.

I apologize for not having caught it in the previous doc ...

**Comment by AlonaKaplan** [comment]
lgtm the hotplug part.

**Review by maiqueb — APPROVED** [review]

File: virt/release_notes/virt-4-15-release-notes.adoc [hunk 2]
```diff
@@ -85,14 +85,14 @@ This release adds new features and enhancements related to the following compone
...
 === Networking
 
-//CNV-30170: NetworkPolicies filtering using IP blocks with OVN-Kubernetes
-
-//CNV-30164: NIC hot plug feature is now GA. This was TP for 4.14. Moved the release note from the TP section to here.
-* You can xref:../../virt/vm_networking/virt-hot-plugging-network-interfaces.adoc#virt-hot-plugging-network-interfaces[hot plug a bridge network interface] to a running virtual machine (VM). Hot plugging and hot unplugging is supported only for VMs created with {VirtProductName} 4.14 or later.
-
-//CNV-30167: SR-IOV NIC hot plug
+//CNV-30164 and CNV-30167: NIC hot plug feature is now GA. This was TP for 4.14. Moved the release note from the TP section to here. Covers both bridge and SR-IOV NICs
+* You can xref:../../virt/vm_networking/virt-hot-plugging-network-interfaces.adoc#virt-hot-plugging-network-interfaces[hot plug a secondary network interface] to a running VM. Hot plugging and hot unplugging is supported only for VMs created with {VirtProductName} 4.14 or later. Hot unplugging is not supported for Single Root I/O Virtualization (SR-IOV) interfaces.
```
Comment (skopacz1) [inline virt/release_notes/virt-4-15-release-notes.adoc:89]: Suggestion to spell out Virtual Machine (especially on the first mention in this section so far):
```suggestion
* You can xref:../../virt/vm_networking/virt-hot-plugging-network-interfaces.adoc#virt-hot-plugging-network-interfaces[hot plug a secondary network interface] to a running virtual machine (VM). Hot plugging and hot unplugging is supported only for VMs created with {VirtProductName} 4.14 or later. Hot unplugging is not supported for Single Root I/O Virtualization (SR-IOV) interfaces.
```

File: virt/release_notes/virt-4-15-release-notes.adoc [hunk 3]
```diff
@@ -85,14 +85,14 @@ This release adds new features and enhancements related to the following compone
...
 
 //CNV-13680: Connecting OVN-Kubernetes to underlying physical networks. Remove known issue BZ-1885605 after adding this note.
+* {VirtProductName} now supports the localnet topology for xref:../../virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc#virt-connecting-vm-to-ovn-secondary-network[OVN-Kubernetes secondary networks]. A localnet topology connects the secondary network to the physical underlay. This enables both east-west cluster traffic and access to services running outside the cluster, but it requires additional configuration of the underlying Open vSwitch (OVS) system on cluster nodes.
+
+//CNV-30170: NetworkPolicies filtering using IP blocks with OVN-Kubernetes
+* An OVN-Kubernetes secondary network is compatible with the xref:../../networking/multiple_networks/configuring-additional-network.adoc#compatibility-with-multi-network-policy_configuring-additional-network[multi-network policy API] which provides the `MultiNetworkPolicy` custom resource definition (CRD) to control traffic flow to and from VMs. You can use the `ipBlock` attribute to define network policy ingress and egress rules for specific CIDR blocks.
```
Comment (skopacz1) [inline virt/release_notes/virt-4-15-release-notes.adoc:95]: Suggestion - I think a comma here might make the long sentence easier to read, up to you:
```suggestion
* An OVN-Kubernetes secondary network is compatible with the xref:../../networking/multiple_networks/configuring-additional-network.adoc#compatibility-with-multi-network-policy_configuring-additional-network[multi-network policy API], which provides the `MultiNetworkPolicy` custom resource definition (CRD) to control traffic flow to and from VMs. You can use the `ipBlock` attribute to define network policy ingress and egress rules for specific CIDR blocks.
```

**Review by skopacz1 — COMMENTED** [review]
Just a few suggestions but otherwise LGTM!

**Comment by sjhala-ccs** [comment]
@Anatw Can you please review the two OVN-K release notes?

**Pull request merged**